Dockerfile.in

```diff
@@ -21,9 +21,4 @@ RUN dnf install -y dnf-plugins-core && \
 COPY augconf /augconf
 RUN augtool -f /augconf
 
-COPY libvirtd.sh /libvirtd.sh
-RUN chmod a+x /libvirtd.sh
-
 RUN for qemu in /usr/bin/qemu-system-*; do setcap CAP_NET_BIND_SERVICE=+eip $qemu; done
-
-CMD ["/libvirtd.sh"]
```

Comment:
If you do this, which to be clear I think is a good idea, you should also tweak `README.md` at the same time to remove all examples of the `kubevirt/libvirt` container being used in a standalone fashion, and to clarify the fact that it is intended *exclusively* as the base image for KubeVirt's virt-launcher container.

Reply:
Okay, I gutted the README